beps/0002-dynamic-frontend-plugins/README.md

```diff
@@ -149,10 +149,57 @@ The outcome of initial testing is positive and it is possible to mix and match d
 
 The experimental code can be found in [this repository](https://github.com/scalprum/mf-mixing-experiments).
 
-**The testing so far was done only on very simple modules**. Although core React features are working (Context API and hooks), more testings needs to be done in order to declare this approach 100% reliable.
-
 So far a lot of custom code needs to be written to bridge Webpack, Rspack, @module-federation/enhanced with Vite. The first three are compatible out of the box, but Vite requires extra bridge to be able to consume/provide modules with/to other builds.
 
+#### React context and singleton sharing
+
+We can share React context and its values. Meaning a shell application (or a plugin parent) can have a context provider and a plugin will consume the context value.
+
+An example is this [package](https://github.com/scalprum/mf-mixing-experiments/tree/master/shared-package) in the experiment repo.
```

Comment:
Noting that we generally discourage contexts that cross plugin boundaries, and when needed use this helper to ensure a global singleton value: https://github.com/backstage/backstage/blob/a5cd12d428b8e701b37946ad1517be5d846e38eb/packages/version-bridge/src/lib/globalObject.ts#L40

That means that we're likely to head down that route rather than relying on federation configuration by integrators when it comes to solving this problem in practice.

That said we still have MUI where we at least need to share the theme context.

Reply:
Yeah, I had the MUI themining in mind. I was interested in this excersise to mainly figure out if we would be limited in any way because of the potential tooling difference. I don't think we are.